sway/commands.c

```diff
@@ -1479,31 +1496,100 @@ static struct cmd_handler handlers[] = {
+        
+        struct bar_config bar = config->bar;
+        int i = list_seq_find(bar.bindings, sway_mouse_binding_cmp_buttons, binding);
+        if (i > -1) {
+		sway_log(L_DEBUG, "bindsym - '%s' for swaybar already exists, overwriting", argv[0]);
+		struct sway_mouse_binding *dup = bar.bindings->items[i];
+		free_sway_mouse_binding(dup);
+		list_del(bar.bindings, i);
+        }
+	list_add(bar.bindings, binding);
+	list_sort(bar.bindings, sway_mouse_binding_cmp);
+
+	sway_log(L_DEBUG, "bindsym - Bound %s to command %s when clicking swaybar", argv[0], binding->command);
+	return cmd_results_new(CMD_SUCCESS, NULL, NULL);
+}
+
+/* static struct cmd_results *bar_cmd_colors(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_font(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_hidden_state(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_id(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_mode(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_modifier(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_output(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_position(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_seperator_symbol(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_status_command(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_strip_workspace_numbers(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_tray_output(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_tray_padding(int argc, char **argv); */
+/* static struct cmd_results *bar_cmd_workspace_buttons(int argc, char **argv); */
+
 static struct cmd_handler bar_handlers[] = {
 	{ "binding_mode_indicator", NULL },
-	{ "bindsym", NULL },
+	/* { "binding_mode_indicator", bar_cmd_binding_mode_indicator }, */
+	{ "bindsym", bar_cmd_bindsym },
 	{ "colors", NULL },
+	/* { "colors", bar_cmd_colors }, */
 	{ "font", NULL },
+	/* { "font", bar_cmd_font }, */
 	{ "hidden_state", NULL },
+	/* { "hidden_state", bar_cmd_hidden_state }, */
 	{ "id", NULL },
+	/* { "id", bar_cmd_id }, */
 	{ "mode", NULL },
+	/* { "mode", bar_cmd_mode }, */
 	{ "modifier", NULL },
+	/* { "modifier", bar_cmd_modifier }, */
 	{ "output", NULL },
+	/* { "output", bar_cmd_output }, */
 	{ "position", NULL },
+	/* { "position", bar_cmd_position }, */
 	{ "seperator_symbol", NULL },
+	/* { "seperator_symbol", bar_cmd_seperator_symbol }, */
 	{ "status_command", NULL },
+	/* { "status_command", bar_cmd_status_command }, */
 	{ "strip_workspace_numbers", NULL },
+	/* { "strip_workspace_numbers", bar_cmd_strip_workspace_numbers }, */
 	{ "tray_output", NULL },
+	/* { "tray_output", bar_cmd_tray_output }, */
 	{ "tray_padding", NULL },
+	/* { "tray_padding", bar_cmd_tray_padding }, */
 	{ "workspace_buttons", NULL },
+	/* { "workspace_buttons", bar_cmd_workspace_buttons }, */
 };
 
+
```

Comment:
I don't like all this commented out code. Just add the code when you implement the thing.